rest/client/client.go

```diff
@@ -29,7 +31,7 @@ func New(apiKey string) Client {
 	c.SetAuthToken(apiKey)
 	c.SetRetryCount(DefaultRetryCount)
 	c.SetTimeout(10 * time.Second)
-	c.SetHeader("User-Agent", "Go client")
+	c.SetHeader("User-Agent", fmt.Sprintf("Go client %v", clientVersion))
```

Comment:
before major release we will want a better user agent that includes `Polygon` or something. And have it be standard across the other client libs

Reply:
agreed, maybe we should change that now? `Polygon Go client v0.7.1`?

Reply:
`Polygon.io GoClient/v0.7.1`
or 
`Polygon.io GoClient/0.7.1`

Reply:
i think i'll go with the first one so it matches the repo tags